tests/product/test_server_install.py

```diff
@@ -351,6 +351,10 @@ def test_install_interactive_with_hostnames(self):
             self.assert_has_default_connector(container)
             self.assert_has_jmx_connector(container)
 
+        # make sure that we don't get an error when using the written out
+        # configs
+        self.run_prestoadmin('server start')
```

Comment:
we should also make sure there isn't an error (though maybe with @ebd2 's change it'll fail out immediately?)